assets/semgrep_rules/client/android-profile-original.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
+  - id: android-profile-original
+    patterns:
+      - pattern-either:
+          - pattern: '....getOriginalProfile()'
+    metadata:
+      author: Brian Johnson
+      source: https://github.com/brave/security-action/blob/main/assets/semgrep_rules/client/android-profile-original.yaml
+      assignees: bridiver
+      category: correctness
+    message: Getting the original profile explicitly can lead to security and privacy issues
```

Comment:
should we advise for a "safer" alternative here?

Reply:
same as below